test/fixture/missing-glyphs.output.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "message": "layers[0].layout.text-field: \"glyphs\" url must be specified",
```

Comment:
:+1: changed to 

 > use of "icon-image" requires a stylesheet "sprite" property
> use of "text-field" requires a stylesheet "glyphs" property